api/src/main/java/org/openmrs/module/fhir/api/helper/ConditionHelper.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.fhir.api.helper;
+
+import org.hl7.fhir.dstu3.model.Condition;
+
```

Comment:
Can we document what this interface is used for?
Also add a since version annotation.

Reply:
done